test/src/org/exist/source/SourceFactoryTest.java

```diff
@@ -134,6 +134,53 @@ public void getSourceFromResource_contextAbsoluteFileUrl_locationRelativeUrl_bas
         assertEquals(getClass().getResource(location), relativeSource.getKey());
     }
 
+    @Test
+    public void getSourceFromResource_contextFolderUrl_locationRelative() throws IOException, PermissionDeniedException {
+        final String contextPath = "resource:org/exist/source";
```

Comment:
what if `final String contextPath = "resource:/org/exist/source";` (note the /)

Reply:
It won't make any difference. However in practice that doesn't happen, we have many tests for this in SourceFactoryTest.java.